src/components/ui/user-autocomplete.tsx

```diff
@@ -0,0 +1,124 @@
+import { CaretDownIcon, CheckIcon } from "@radix-ui/react-icons";
+import { PopoverClose } from "@radix-ui/react-popover";
+import { useTranslation } from "react-i18next";
+
+import { cn } from "@/lib/utils";
+
+import { Button } from "@/components/ui/button";
+import {
+  Command,
+  CommandEmpty,
+  CommandGroup,
+  CommandInput,
+  CommandItem,
+  CommandList,
+} from "@/components/ui/command";
+import {
+  Popover,
+  PopoverContent,
+  PopoverTrigger,
+} from "@/components/ui/popover";
+
+import { Avatar } from "@/components/Common/Avatar";
+
+import { formatName, isUserOnline } from "@/Utils/utils";
+import { UserBase } from "@/types/user/user";
+
+interface UserCompleteProps {
+  id: string;
+  disabled?: boolean;
+  value: UserBase | undefined;
+  isLoading?: boolean;
+  onSelect: (user: UserBase | null) => void;
+  options: {
+    users: UserBase[];
+  };
+}
+
+export default function UserAutocomplete({
+  value,
+  options,
+  onSelect,
+  isLoading,
+}: UserCompleteProps) {
+  const { t } = useTranslation();
+
+  return (
+    <Popover>
+      <PopoverTrigger asChild disabled={isLoading}>
+        <Button
+          variant="outline"
+          role="combobox"
+          className="min-w-60 justify-start"
+        >
+          {value ? (
+            <div className="flex items-center gap-2">
+              <Avatar
+                imageUrl={value.profile_picture_url}
+                name={formatName(value)}
+                className="size-6 rounded-full"
+              />
+              <span>{formatName(value)}</span>
+            </div>
+          ) : (
+            <span>{t("show_all")}</span>
+          )}
```

Comment:
the fallback: "show all" doesn't make sense in Forms, only on filters right?

let's take placeholder from props instead